src/PowerShellEditorServices/Debugging/StackFrameDetails.cs

```diff
@@ -58,19 +74,21 @@ public class StackFrameDetails
         /// </param>
         /// <returns>A new instance of the StackFrameDetails class.</returns>
         static internal StackFrameDetails Create(
-            CallStackFrame callStackFrame,
+            PSObject callStackFrameObject,
             VariableContainerDetails autoVariables,
             VariableContainerDetails localVariables)
         {
             return new StackFrameDetails
             {
-                ScriptPath = callStackFrame.ScriptName ?? "<No File>",
-                FunctionName = callStackFrame.FunctionName,
-                LineNumber = callStackFrame.Position.StartLineNumber,
-                ColumnNumber = callStackFrame.Position.StartColumnNumber,
+                ScriptPath = (callStackFrameObject.Properties["ScriptName"].Value as string) ?? NoFileScriptPath,
```

Comment:
Definitely, if there turns out to be a good reason for it we can make the change.